### DISCUSSION TITLE
PK-Sim AUC_tEnd calculating method

### DISCUSSION THREAD

**Discussion by akim2000** [Open-Systems-Pharmacology/Forum#1782] [Questions & Problems]
I wanted to compare AUCs for some of my models that I am building in MoBi vs. the base PK-Sim model. In order to do so as accurately as possible, I was wondering what method PK-Sim uses in its PK-Analysis to calculate the AUC_tEnd value for the model simulation curve, so that I can apply the same method to my simulation results in MoBi to determine the AUC, since MoBi does not have a built-in PK-Analysis (which I think would be a great addition, unless it already exists and I just happened to miss it completely).

**Comment by wilbertdew** [comment]
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-simulations#pk-analysis-view

An alternative for your problem is: save the mobi simulation as .pkml file, import it in PK-Sim, do the PK-Analysis there
